### DISCUSSION TITLE
Customise Schedule Embed Widget

### DISCUSSION THREAD

**Discussion by Dedras** [AzuraCast/AzuraCast#7184] [Q&A]
Hi there.
Need to change colors in Schedule Embed Widget, any help would be really appreciated.

**Comment by LordHelmchen666** [comment]
You can add in your station custom CSS something like this
```

/* schedule css */
root, [data-bs-theme="light"] {
  --bs-link-color: #650d88;
  --bs-link-color-rgb: 101, 13, 136;
  --bs-primary-rgb: 101, 13, 136;
}
root, [data-bs-theme="dark"] {
  --bs-link-color: #fca103;
  --bs-link-color-rgb: 252, 161, 3;
  --bs-primary-rgb: 101, 13, 136;
}
:root {
	--fc-event-bg-color: #650d88;
	--fc-event-border-color: #440040;
	--fc-highlight-color: rgba(188,232,241,.3);
	--fc-today-bg-color: #e782004f;
}
.fc .fc-timegrid-now-indicator-line {
	border-style: dashed;
	border-width: 2px 0px 0px;
}
.fc-direction-ltr .fc-timegrid-now-indicator-arrow {
	border-width: 9px 0px 9px 12px;
}
/* End schedule css */

```

This are not all possible values! But most important for me. :)

**Comment by Dedras** [comment]
Hi there
Thanks for your time to help.

When I copy your full code everything works just fine but when I change the HEXA codes, only  the --fc-today-bg-color:  works.

By the way any idea how to remove the header or top row?

**Reply by LordHelmchen666** [reply]
What exactly don't work? show the code please.

**Comment by Dedras** [comment]
:root {
	--fc-event-bg-color: #54595F;                         (It does not work)
	--fc-event-border-color: #54595F;                  (It does not work)
	--fc-highlight-color:  rgba(84, 89, 95, 1;          (It does not work)
	--fc-today-bg-color: #F7EE7A;                        (I works)
}

**Reply by LordHelmchen666** [reply]
You forgot a )
```
--fc-event-bg-color: #54595F;
--fc-event-border-color: #54595F;
--fc-highlight-color: rgba(84, 89, 95, 1); 
--fc-today-bg-color: #F7EE7A; 
```
Other colors are working for me. Any cache involved?

**Comment by Dedras** [comment]
I just made a new instalation, same problem, only --fc-today-bg-color: #F7EE7A; works for my HEXA codes, your HEXA codes work all just fine.
Where should I use a) ?
Any idea how to remove the header or top row?
